packages/sdk/src/oracle/services/tests/erc20.e2e.ts

```diff
@@ -3,11 +3,11 @@ import assert from "assert";
 import { ethers } from "ethers";
 import { Provider } from "@ethersproject/providers";
 import { factory, Erc20 } from "../erc20";
-import { MULTICALL2_ADDRESS } from "../../constants";
 
 dotenv.config();
 
 const wethAddress = "0x7355Efc63Ae731f584380a9838292c7046c1e433";
+const multicall2Address = process.env.multicall2Address;
```

Comment:
If you're already assuming the mainnet WETH address, is there any harm in assuming the mainnet multicall address?

Reply:
yea can do that